lib/read_only/lfric/Makefile

```diff
@@ -80,27 +80,21 @@ read_only.o: 	  $(LFRIC_LIB) read_only.f90 read_only_base.o
 
 read_only_base.o: psy_data_base.o
 
-
-# Rules for file creation:
-# ------------------------
-read_only.f90:	read_only.jinja Makefile
-	$(PROCESS) $(PROCESS_ARGS) read_only.jinja > read_only.f90
-
 %.o: %.f90
 	$(F90) $(F90FLAGS) -c $<
 
 # The LFRic ReadOnly library is built on the psy-data base library
 # and the read-only base library. So add the rules to process
 # these files to get the plain Fortran files:
 psy_data_base.f90: ../../psy_data_base.jinja Makefile
-	$(PROCESS) $(PROCESS_ARGS) $< > psy_data_base.f90
+	$(PROCESS) $(PROCESS_ARGS) -generic-declare $< > psy_data_base.f90
```

Comment:
Please extend the comment to explain what the `-generic-xxx` arguments are for.

Reply:
Done.